extensions-core/lookups-cached-global/src/main/java/io/druid/query/lookup/namespace/JDBCExtractionNamespace.java

```diff
@@ -55,10 +55,8 @@ public JDBCExtractionNamespace(
       final MetadataStorageConnectorConfig connectorConfig,
       @NotNull @JsonProperty(value = "table", required = true)
       final String table,
-      @NotNull @JsonProperty(value = "keyColumn", required = true)
-      final String keyColumn,
-      @NotNull @JsonProperty(value = "valueColumn", required = true)
```

Comment:
Perhaps we should have a "default map" name that the keyColumn/valueColumn go into if you don't specify a `maps` list. And then that one also gets used at query time if you don't specify a mapName.